Source/ACE.Entity/ObjectGuid.cs

```diff
@@ -5,14 +5,71 @@ namespace ACE.Entity
     public enum GuidType
     {
         Undef,
+        Weenie,
+        Static,
+        Generator,
         Player,
-        Creature
+        Creature,
+        Item
     }
 
     public struct ObjectGuid
     {
         public static readonly ObjectGuid Invalid = new ObjectGuid(0);
 
+        public static uint WeenieMin { get; } = 0x00000001;
+        public static uint WeenieMax { get; } = 0x000F423F;
+
```

Comment:
the weenie range is probably reasonably good to stay between 1 and 999,999. Turbine only had at most 60,000 weenies in the entire run.

The generator range I added is altogether very likely way to small. It needs to account for generators added to the database as well as cloned/spawned generator copies as we add camp spawning, etc.. I think at this point, it's fine to just start with a smaller range to start until we get a plan in place for reassignment as something like this is pool usage is going to burn through ids as landscape spawns come and go.

As is now, we only have about 20 generators in our DB and they aren't near finished in design yet